src/mars.d

```diff
@@ -101,6 +101,7 @@ Where:
 <option>:
   @<cmdfile>       read arguments from cmdfile
   -allinst         generate code for all template instantiations
+  -bc-ctfe         use the experimental bytecode-engine for ctfe
```

Comment:
Well it's not a transition CTFE should work exactly as before.
I do not think it harms anything approving the switch.